src/config.cc

```diff
@@ -95,8 +95,8 @@ Config::Config() {
   };
   FieldWrapper fields[] = {
       {"daemonize", true, new YesNoField(&daemonize, false)},
-      {"bind", true, new StringField(&binds_, "127.0.0.1")},
```

Comment:
We should not remove this line?

Reply:
It seems a bug in github, I do not remove this line.

https://github.com/apache/incubator-kvrocks/pull/797/files#diff-76602be0b8dbf09757c540e3b4c474b477cac3bdf38ca318e6284cb34020cd2eR97-R105

Reply:
@PragmaTwice Yes, that's weird. It only can be seen in last commit, let ignore this.